pkg/scheduler/scheduler.go

```diff
@@ -653,6 +653,7 @@ func (s *Scheduler) requeueAndUpdate(ctx context.Context, e entry) {
 	if e.status == notNominated || e.status == skipped {
 		patch := workload.BaseSSAWorkload(e.Obj)
 		workload.AdmissionStatusPatch(e.Obj, patch, true)
+		workload.AdmissionChecksStatusPatch(e.Obj, patch, s.clock)
```

Comment:
This looks sensible, but it would be great to repro the issue integration tests (we could probably use them to simulate the external admission check). cc @PBundyra